Assets/HoloToolkit/Utilities/Scripts/DebugPanelControllerInfo.cs

```diff
@@ -104,6 +134,40 @@ private string GetControllerInfo()
                                           controllerState.MenuPressed, controllerState.SelectPressed, controllerState.SelectPressedAmount,
                                           controllerState.ThumbstickPressed, controllerState.ThumbstickPosition, controllerState.TouchpadPressed,
                                           controllerState.TouchpadTouched, controllerState.TouchpadPosition);
+
+                // Text label display
+                if(controllerState.Handedness.ToString().Equals("Left"))
```

Comment:
It'd probably be good to check controllerState.Handedness.Equals(InteractionSourceHandedness.Left) here instead of a string comparison (and the right controller below).